src/pages/job-opening-page.ts

```diff
@@ -0,0 +1,169 @@
+import { customElement, observe, property } from '@polymer/decorators';
+import { html, PolymerElement } from '@polymer/polymer';
+import '@power-elements/lazy-image';
+import { RouterLocation } from '@vaadin/router';
+import '../components/hero/simple-hero';
+import '../components/text-truncate';
+import '../elements/content-loader';
+import '../elements/shared-styles';
+import { router } from '../router';
+import { ReduxMixin } from '../store/mixin';
+import { heroSettings, jobs } from '../utils/data';
+import { updateImageMetadata } from '../utils/metadata';
+
+@customElement('job-opening-page')
+export class JobOpeningPage extends ReduxMixin(PolymerElement) {
+
+  static get template() {
+    return html`
+        <style include="shared-styles flex flex-alignment positioning">
+        :host {
+          display: block;
+          height: 100%;
+        }
+        .job-view-container {
```

Comment:
.job-view-container { ???
Rename please

PLEASE SEARCH IN PROJECT AND REPLACE